public/ng-views/maps.html

```diff
@@ -394,36 +395,100 @@ <h6 ng-repeat="(key, variant) in variants" ng-show="variant.sel">
 </leaflet>
 <leaflet  center   = "center_right"
           defaults = "defaults_right"
-          maxbounds = "maxBounds_right"
           id       = "map_right"
           tiles    = "tiles_right"
           >
 </leaflet>
 
 
-<div class="basecol right" ng-show="overview" style="max-height: 80%">
+<!--CONTROL PANEL ROUTE SELECTOR MODULE--> 
+
+<div class="basecol_routeSelector col-md-2" >
+  <span>
+  <div class="fa-stack ">
+    <i class="fa fa-circle fa-stack-2x"></i>
+    <i class="fa fa-bus fa-stack-1x fa-inverse"></i>
+  </div> 
+  </span>
+
+  <!-- <span ng-show="dotSelect"> -->
+  <span>
+    <span class="fa-stack" 
+          ng-click="targetCorridor('BH'); 
+                    tabnav='BH'; overview=true"
+          ng-class="tabnav !== 'BH' ? 'dot-unselected' : ''"
+          style="color:#0071CA">
+      <i class="fa fa-circle fa-stack-2x"></i>
+      <i class="fa fa-stack-1x fa-inverse">
+        <small>BH</small>
+      </i>
+    </span>
+
+    <span class="fa-stack" 
+          ng-click="targetCorridor('HP'); 
+                    tabnav='HP'; overview=true"
+          ng-class="tabnav !== 'HP' ? 'dot-unselected' : ''"
+          style="color:#B10086">
+      <i class="fa fa-circle fa-stack-2x"></i>
+      <i class="fa fa-stack-1x fa-inverse">
+        <large>HP</small>
+      </i>
+    </span>
+
+    <span class="fa-stack" 
+          ng-click="targetCorridor('HD'); 
+                    tabnav='HD'; overview=true"
+          ng-class="tabnav !== 'HD' ? 'dot-unselected' : ''"
+          style="color:#CE5504">
+      <i class="fa fa-circle fa-stack-2x"></i>
+      <i class="fa fa-stack-1x fa-inverse">
+        <small>HD</small>
+      </i>
+    </span>
+
+    <span class="fa-stack" 
+          ng-click="targetCorridor('CT'); 
+                    tabnav='CT'; overview=true"
+          ng-class="tabnav !== 'CT' ? 'dot-unselected' : ''"
+          style="color:#AD0101">
+      <i class="fa fa-circle fa-stack-2x"></i>
+      <i class="fa fa-stack-1x fa-inverse">
+        <small>CT</small>
+      </i>
+    </span>
+  </span>
+</div>  
+
+<!--END OF ROUTE SELECTOR MODULE--> 
+
+
+<!--ROUTE TABLE MODULE--> 
+
+<div class="basecol_routeTable"  > 
+
+<!-- <div class="basecol right" ng-show="overview" style="max-height: 80%">
   <h4 ng-click="overview=false; tabnav=null">
     Routes Overview
     <i class="fa fa-times-circle-o pull-right"></i>
-  </h4>
-
+  </h4> -->
+  
   <div class="col-xs-12 body">
 
-    <h5 ng-show="variants[tabnav].sel">
+    <h6 ng-show="variants[tabnav].sel">
       <span class="badge pull-right" style="color: #{{ routes[variants[tabnav].all[variants[tabnav].sel].routeId][0].options.base.routeColor }}"
-            ng-click="updateRouteScorecard(scenario[tabnav].routeId, tabnav); routeScorecard=true">
+            ng-click="updateRouteScorecard(scenario[tabnav].routeId, tabnav); routeScorecard=!routeScorecard">       
         Route Scorecard
       </span>
 
-      <span>
+      <span style="font-size: 1em">
```

Comment:
do not set fonts inline. also do not set font size by em unless you are correctly cascading styles - right now you are inheriting em vals from an unknown because you have not been consistent about employing this